client/client.go

```diff
@@ -166,8 +169,15 @@ var (
 	noServersErr = errors.New("no servers")
 )
 
+// ClientTelemetry is a subset of global telemetry configuration options that
+// are relevant for the client
+type ClientTelemetry struct {
```

Comment:
Remove this and just add the telemetry block into the client configuration. Then the agent can just assign it when creating the client config. In the future client configs should go in the client/config/config.go